pravega/pravega-client-rust
# Issue 363: Nodejs publish

**Change log description**  
Build and publish the Nodejs bindings for different systems and architectures.

**Purpose of the change**  
Fix #363 

**What the code does**  
It basically pretties the docs and updates CI so npm packages are published to the npm registry and platform-specific binaries are released at Github releases pages.
For more details about the release process, see https://github.com/neon-bindings/neon/issues/117#issuecomment-1073561462

This should be merged after #360.

**How to verify it**  
`tagPublish` workflow will publish npm packages and platform-specific binaries are released at Github releases pages.
See my own fork's actions and release page for the expected outcome.

https://www.npmjs.com/package/@thekingofcity/pravega
https://github.com/thekingofcity/pravega-client-rust/actions/runs/2014604184
https://github.com/thekingofcity/pravega-client-rust/releases/tag/v0.4.12

Release of this tag must be created so binaries can be uploaded!
